Intersect.Client/Core/Graphics.cs

```diff
@@ -41,6 +41,8 @@ public static FloatRect CurrentView
                 Renderer.SetView(_currentView);
             }
         }
+        
+        public static FloatRect Viewport => new FloatRect(CurrentView.Position, CurrentView.Size / (Globals.Database?.WorldZoom ?? 1));
```

Comment:
Can you change this to WorldViewport so this isn't confused when UI Scaling is eventually introduced?

Reply:
Done